resources/assets/coffee/_classes/beatmap-discussion-helper.coffee

```diff
@@ -61,3 +74,14 @@ class @BeatmapDiscussionHelper
       praise: '&#xf004;'
       suggestion: '&#xf10c;'
       problem: '&#xf06a;'
+      resolved: '&#xf05d;'
+
+
+  @moderationGroup: (user) =>
+    if user.groups?
+      _.intersection(user.groups, ['admin', 'qat', 'bng'])[0]
+    else
+      switch
+        when user.isAdmin then 'admin'
+        when user.isQAT then 'qat'
+        when user.isBNG then 'bng'
```

Comment:
the other option is to display "BN" but give these tooltips

qat -> Quality Assurance Team
bn -> Beatmap Nominator